test/js/flicking.test.js

```diff
@@ -1224,9 +1224,6 @@ test("When intent to initialize with non-existent element, should throw error.",
 		return true;
 	});
 
-	// Given
-	$(document.body).append("<div id='no-child'></div>");

```

Comment:
To get understanding of the given element, it's better stay here rather than appending in html file.
Just giving space is enough to give childNode.
```js
$(document.body).append("<div id='no-child'> </div>");
```